padd.sh

```diff
@@ -441,6 +441,9 @@ GetVersionInformation() {
     docker_version_converted="$(VersionConverter "${DOCKER_VERSION}")"
     docker_version_latest_converted="$(VersionConverter "${GITHUB_DOCKER_VERSION}")"
 
+    # Note: the version comparison will fail for any Docker tag not following a 'vYYYY.MM.VV' scheme
```

Comment:
The correct tag format is `YYYY.MM.VV`, without the initial `v`.

I think I did the same mistake when I wrote the comments on MM.

Reply:
I push-forced it away